holmes/plugins/toolsets/newrelic/newrelic.py

```
@@ -0,0 +1,127 @@
+            return False, "No configuration provided"
+
+        try:
+            nr_config = NewrelicConfig(**config)
+            self.nr_account_id = nr_config.nr_account_id
+            self.nr_api_key = nr_config.nr_api_key
+            self.is_eu_datacenter = nr_config.is_eu_datacenter or False
+
+            if not self.nr_account_id or not self.nr_api_key:
+                return False, "New Relic account ID or API key is missing"
+
+            return True, None
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)